RA_Integration/RA_Dlg_Memory.cpp

```diff
@@ -793,51 +800,6 @@ INT_PTR Dlg_Memory::MemoryProc( HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPar
 {
 	switch ( nMsg )
 	{
-		case WM_TIMER:
```

Comment:
timer was eliminated, search results are updated each time the memory viewer is updated